src/components/SearchToolbar/AccessibilityFilterMenu.js

```diff
@@ -0,0 +1,216 @@
+    toiletFilter: ['yes'],
+  },
+  fully: {
+    // translator: Button caption in the filter toolbar. Answer to the question 'which places you want to see'
+    caption: t`Only fully wheelchair accessible`,
+    accessibilityFilter: ['yes'],
+    toiletFilter: [],
+  },
+  fullyWithWC: {
+    // translator: Button caption in the filter toolbar. Answer to the question 'which places you want to see'
+    caption: t`Only fully wheelchair accessible + WC`,
+    accessibilityFilter: ['yes'],
+    toiletFilter: ['yes'],
+  },
+  unknown: {
+    // translator: Button caption in the filter toolbar. Answer to the question 'which places you want to see'
+    caption: t`Places that I can contribute to`,
+    accessibilityFilter: ['unknown'],
+    toiletFilter: [],
+  },
+  notAccessible: {
+    // translator: Checkbox caption on the filter toolbar. If the checkbox is clicked, only places that are not wheelchair accessible are shown.
+    caption: t`Only places that are not accessible`,
+    accessibilityFilter: ['no'],
+    toiletFilter: [],
+  },
+};
+
+
+function findFilterKey({ toiletFilter, accessibilityFilter }) {
+  return Object.keys(availableFilters).find(key => {
+    const filter = availableFilters[key];
+    const requestedToiletFilter = isEqual(toiletFilter, ['yes', 'no', 'unknown']) ? [] : toiletFilter;
+    return isEqual(requestedToiletFilter, filter.toiletFilter) && isEqual(accessibilityFilter.sort(), filter.accessibilityFilter.sort())
+  });
+}
+
+
+class AccessibilityFilterMenu extends React.Component<Props, State> {
+  static defaultProps: DefaultProps;
+  toolbar: ?React.ElementRef<typeof Toolbar>;
+  toiletCheckbox: ?React.ElementRef<'input'>;
+  closeButton: ?React.ElementRef<typeof CloseButton>;
+
+  state = {
+    toiletCheckboxFocused: false,
+  };
+
+  render() {
+    const { accessibilityFilter, toiletFilter } = this.props;
+    const category = this.props.category || 'undefined';
+    const currentFilterKey = findFilterKey({ accessibilityFilter, toiletFilter });
+    const shownFilterKeys = currentFilterKey ? [currentFilterKey] : Object.keys(availableFilters);
+    const lastIndex = shownFilterKeys.length - 1;
+
+    return (
+      <section
+        className={this.props.className}
+        aria-label={t`Wheelchair Accessibility Filter`}
+      >
+        <section className="accessibility-filter">
+          {shownFilterKeys.map((key, index) => (
+            <AccessibilityFilterButton
+              accessibilityFilter={availableFilters[key].accessibilityFilter}
+              toiletFilter={availableFilters[key].toiletFilter}
+              caption={availableFilters[key].caption}
+              category={category}
+              isMainCategory
+              isActive={currentFilterKey}
+              showCloseButton={shownFilterKeys.length === 1}
+              onKeyDown={({nativeEvent}) => {
+                const tabPressedOnLastButton = index === lastIndex && nativeEvent.key === 'Tab' && !nativeEvent.shiftKey;
```

Comment:
Maybe: move into its own method?

Reply:
Even better: Not needed anymore because tab handling does the right thing here.